Writerside/topics/conclusion-and-outlook.md

```diff
@@ -1,3 +1,28 @@
 # Conclusion and Outlook
 
-Concluding it can be stated, that all must-have requirements are fully satisfied.
\ No newline at end of file
+In conclusion, the project successfully fulfilled all the must-have, most of the should-have, and some of the
+nice-to-have requirements, delivering a production-ready transit routing solution.
+
+One of the key strengths of this project is that the final product is available as open-source under the MIT license,
+which encourages future contributions and improvements. Additionally, the team’s strong motivation were vital to our
+success, resulting in a productive development process enriched by a compelling and challenging topic. The iterative
+approach we adopted, along with careful task management, ensured efficiency, even with extensive pull request reviews.
+This not only maintained high code quality but also resulted in solid design decisions and robust testing practices,
+which proved beneficial in the long run. Each team member expanded their expertise in key areas, including data
+structures and algorithms, language specifics, and software architecture, contributing to substantial knowledge gains.
+
+However, there were areas for improvement. One notable challenge was the minimal overlap between the C++ and Java
```

Comment:
maybe also add the poor implementation of GTFS by the national transport organizations.

Reply:
What exactly do you mean by poor? 

Missing accessibility information? Incomplete transfers between stops (parent / child)?

Reply:
Suggestion in: https://github.com/naviqore/documentation/pull/25/commits/a09551a98ffaf6998e867a92b085491d3b981d40